src/components/XchangeCourseTable.js

```diff
@@ -1,19 +1,46 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import {
   Table,
   TableBody,
   TableCell,
   TableContainer,
   TableHead,
+  TablePagination,
   TableRow,
   TextField,
 } from "@mui/material";
 
 function XchangeCourseTable(props) {
-    const search = (query) => {
-        console.log(query);
-    };
+  const [courseEquivalencies, setCoursesEquivalency] = useState([]);
```

Comment:
This modifies the XchangeCourseTable Component which is used by other pages rather than sending the component the table body props argument that it populates the page with. 

Move this code into the Course Page and then use props to send your table body into the component.

Reply:
So this is actually the only table in the whole app that has a textfield baked into it, so I think XChangeCourseTable actually isn't going to be used elsewhere. 
<img width="531" alt="image" src="https://github.com/nehageorge/xchange/assets/50978011/ccaec068-1b76-4b3b-a645-878b4603facf">

The embedded textfield is also why I'm passing the course equivalency data into the table itself. I think I need to have the textfield be able to filter the course equivalency data which then determines the table body, which can't happen if I just pass it into the Course page (since the textfield is within the XChangeCourseTable).

Lmk if that makes sense or if there's something else I could do different

Reply:
From the review views, it looked like a reusable component that took `props.tableBody` was modified.

If this is a new component that you made that is not reusable then in that case this can be left as is. 😄

Reply:
Oh true that line should be removed that's my b

Reply:
Yeah it's not reusable